.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,70 @@
+name: Bug report
+description: Create a bug report to help us improve. Use this template if you encountered an issue while integrating with or implementing the APIs of this SDK.
+labels: [bug, triage-required]
+
+body:
+- type: checkboxes
+  attributes:
+    label:  Before logging a bug report please confirm the following
+    description: Go through the items below before logging an issue.
+    options:
+    - label: This is not a Security Disclosure, otherwise please follow the guidelines in [Security Policy](https://github.com/adobe/aepsdk-edge-ios/security/policy).
+      required: true
+    - label: I have searched in this repository's issues to see if it has already been reported.
+      required: true
+    - label: I have updated to the latest released version of the SDK and the issue still persists.
+      required: true
+
+- type: textarea
+  attributes:
+    label: Bug summary
+    description: Please provide a summary of the bug you are reporting.
+  validations:
+    required: true
+

```

Comment:
Wonder if we would benefit from knowing how frequently a bug is being seen (like assigning priority)

Reply:
For frequently asked questions/bugs, on other open source repos they usually get consolidated in a single issue and others are linked and closed as duplicates, and usually the users will search for an existing issue and reply in thread. We can do the deduplication thing at triage time.
I do not see a need for priority at this time, but we can always reevaluate this as we see more questions coming in through this form.

Reply:
I was mainly referring to "frequency" / "reproducibility" as a field. But yea aligned with adding them later.